testdata/controllers/config.yaml

```diff
@@ -0,0 +1,23 @@
+defaultResync: 30s
+# by default, Iter8 controller is namespace scoped
+# clusterScoped: true
+resourceTypes:
+  deploy:
+    Group: apps
+    Version: v1
+    Resource: deployments
+  svc:
+    Group: ""
+    Version: v1
+    Resource: services
+  isvc:
+    Group: serving.kserve.io
+    Version: v1beta1
+    Resource: inferenceservices
+    conditions:
+    - name: Ready
+      status: "True"
+  vs:
+    Group: networking.istio.io
+    Version: v1beta1
+    Resource: virtualservices
```

Comment:
@Alan-Cha points out that we had initial suggested that `group`, `version` and `resource` should be lower case. I observe that the controller code is independent of case. However, the helm chart seems to be dependent. Clearly, it would be good to make a helm chart that handles both. In the meantime, can we make these lower case?

Reply:
schema.GroupVersionResource is directly embedded in our GVRK struct.

Unfortunately, `schema.GroupVersionResource` has no JSON tags, so they are capitalized.
***
For now, I suggest we retain this approach. The Helm chart can make the assumption that GVR is capitalized, it doesn't need to assume it is small.